lib/mutant/cli/command/run.rb

```diff
@@ -153,6 +153,9 @@ def add_runner_options(parser)
           parser.on('-j', '--jobs NUMBER', 'Number of kill jobs. Defaults to number of processors.') do |number|
             set(jobs: Integer(number))
           end
+          parser.on('-t', '--mutation-timeout NUMBER', 'Per mutation analysis timeout') do |number|
+            set(mutation_timeout: Float(number))
```

Comment:
Argument coercions via `Float` currently raise uncaught exceptions, optparse has a better API, but I chose to use this one for consistency with `--jobs` above. A fix will need to be made globally.